adcm_aio_client/core/objects/cm.py

```diff
@@ -341,6 +348,14 @@ class HostsAccessor(PaginatedAccessor[Host, None]):
     class_type = Host
 
 
+class HostsNode(HostsAccessor):
+    async def add(self: Self, provider: HostProvider, name: str, cluster: Cluster | None = None) -> None:
+        data = {"hostproviderId": provider.id, "name": name}
+        if cluster:
+            data["clusterId"] = cluster.id
+        await self._requester.post(*self._path, data=data)
```

Comment:
always use `get_own_path`, not implementation details, pls

Reply:
It's a node. It doesn't have get_own_path